src/main/java/seedu/kitchenhelper/command/AddInventoryCommand.java

```diff
@@ -0,0 +1,102 @@
+package seedu.kitchenhelper.command;
+
+import seedu.kitchenhelper.object.Chore;
+import seedu.kitchenhelper.object.Recipe;
+import seedu.kitchenhelper.object.ingredient.Ingredient;
+import seedu.kitchenhelper.object.ingredient.Dairy;
+import seedu.kitchenhelper.object.ingredient.Drink;
+import seedu.kitchenhelper.object.ingredient.Fruit;
+import seedu.kitchenhelper.object.ingredient.Meat;
+import seedu.kitchenhelper.object.ingredient.Miscellaneous;
+import seedu.kitchenhelper.object.ingredient.Staple;
+import seedu.kitchenhelper.object.ingredient.Vegetable;
+
+import java.util.ArrayList;
+
+/**
+ * Adds the ingredient to the inventory list.
+ */
+public class AddInventoryCommand extends Command {
```

Comment:
got to check with the team about which coding format we are using bur looks good!